command/command_test.go

```diff
@@ -266,7 +267,7 @@ func testState() *states.State {
 				Type: "test",
 			}.Absolute(addrs.RootModuleInstance),
 		)
-	})
+	}).DeepCopy()
```

Comment:
This `DeepCopy` made me :thinking: a bunch because there's no other variable holding a reference to the state we built here and thus the copying seems superfluous.

However, reading down to see the other changes below I assume the goal here is to make sure that all of the data in the state makes it into the copy, and that makes sense. For the benefit of a future maintainer who won't have the benefit of seeing this line in the context of this PR diff, perhaps a short comment explaining that this is here to implicitly test `DeepCopy` would be helpful so it doesn't get cleaned up.